src/main/java/core/basesyntax/supplier/ColorSupplier.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.supplier;
+
+import core.basesyntax.constant.Color;
+import java.util.Random;
+
+public class ColorSupplier {
+    private final Random random = new Random();
+
+    public String getRandomColor() {
+        Color[] colors = Color.values();
+        int index = random.nextInt(colors.length);
+        Color color = colors[index];
+        return color.name();

```

Comment:
`color` var use only once, thus can be removed
 ```suggestion
         return colors[index].name();
```